src/components/MultiButtonWithSubQuery.tsx

```diff
@@ -0,0 +1,106 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React from 'react';
+
+import { Button } from './Button';
+
+export type TSubQueryType = {
+  buttons?: [{ buttonTitle: string; routeName: string; webUrl: string }];
+  buttonTitle: string;
+  params: { rootRouteName: string; routeName: string; title: string; webUrl: string };
+  routeName: string;
+  webUrl?: string;
+};
```

Comment:
this should go to types as separate fie.
importing a type from components seems to be misplaced.

![image](https://user-images.githubusercontent.com/1942953/203080772-8108e36d-727f-4024-9651-975910a9e075.png)

i will refactor that.